websites/X/Xero.gg/presence.ts

```diff
@@ -14,20 +14,27 @@ presence.on("UpdateData", async () => {
     largeImageKey: "xero"
   };
 
+  const showChat = await presence.getSetting("showchat"),
```

Comment:
bruh yeah, combine it with presenceData const (i didn't saw it because it was hidden)

Reply:
OOOOOH, makes sense, I didn't notice it either...